src/components/TokenHolderList/TokenHolderList.tsx

```diff
@@ -30,7 +30,10 @@ function TokenHolderList({ title, tokenHoldersRef }: Props) {
     nonNullTokenHolders.sort((a, b) => {
       const usernameA = a.user.username.toLowerCase();
       const usernameB = b.user.username.toLowerCase();
+
       return (
+        // TODO(Terence): What is going on here
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
CC @kaitoo1 we're subtracting booleans?

Reply:
classic js magic operation so we can update to make it legible and typescript compliant